### PR TITLE
Added "Support Development" feature in settings screen

### LINKED ISSUE #69: [FEATURE] Adding more setting options in Settings screen
### Description

The Settings Screen of the application currently has only 1 setting option: the share app option. I'd add more setting options to the app, such as 
- Turning notifications on or off
- Language
- Support Development: This would bring users to the Vritant repository's issues section, highlighting concerning issues to contribute to. 

### Screenshots

Screenshot_20230730_074015

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jaivsh**
## Description 

I've added the support development feature, clicking on which the user would be redirected to the ```issues``` section of the Vritant's github repository, where he/she would be able to see all the existing issues related to the application. 

Close #69 

## Screenshots
Screenshot_20230801_100140

### CODE REVIEW

**Comment by jaivsh** [comment]
@bhattabhi013 please review this PR

File: pubspec.lock [hunk 1]
```diff
@@ -25,6 +25,14 @@ packages:
...
     source: hosted
     version: "0.7.3"
+  args:
+    dependency: transitive
+    description:
+      name: args
+      sha256: eef6c46b622e0494a36c5a12d10d77fb4e855501a91c1b9ef9339326e58f0596
+      url: "https://pub.dev"
+    source: hosted
+    version: "2.4.2"
```
Comment (bhattabhi013) [inline pubspec.lock:35]: revert these changes as they are not required

**Review by bhattabhi013 — CHANGES_REQUESTED** [review]

File: pubspec.lock (hunk 1 above)
Comment (jaivsh) [reply pubspec.lock:35]: Done @bhattabhi013

**Comment by jaivsh** [comment]
@bhattabhi013 I've resolved the isue pertaining to the pubspec.yaml. Can it be merged now?

**Review by bhattabhi013 — APPROVED** [review]
LGTM 🚀

**Comment by bhattabhi013** [comment]
Please resolve the conflicts

**Comment by jaivsh** [comment]
@bhattabhi013 I've resolved conflicts, please check

**Review by bhattabhi013 — APPROVED** [review]
